core/changedetector/change_detector.go

```diff
@@ -10,6 +10,8 @@ type ChangeRequest struct {
 	OldData     []byte
 	NewData     []byte
 	Depth       int32
+	SourceUrl   string
```

Comment:
`SourceUrl` -> `SourceURL`

Reply:
might have missed as taken commitSHA. Fixed it